File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    arr = []
    words = s.split(' ')
    for word in words:
        if '-' in word:
            word = word[1::]
            arr.append(int(word) * -1)
        else:
            arr.append(int(word))
    arr.sort()
    answer = str(arr[0]) + ' ' + str(arr[-1])
    return answer

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <sstream>
#include <algorithm>

using namespace std;

string solution(string s) {
    string answer = "";
    vector<int> arr;
    stringstream ss(s);
    string token;
    
    while (ss >> token){
        int num = stoi(token);
        arr.push_back(num);
    }
    
    int mx = *max_element(arr.begin(), arr.end());
    int mn = *min_element(arr.begin(), arr.end());
    
    return to_string(mn) + " " + to_string(mx);
}